modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md

```diff
@@ -188,8 +188,8 @@ For this purpose, the typeclass [`MonadError`]({{ '/docs/arrow/typeclasses/monad
 [`MonadError`]({{ '/docs/arrow/typeclasses/monaderror' | relative_url }}) allows us to raise and recover from errors.
 
 ```kotlin
-fun getLineLengthAverage(path: FilePath): IO<List<String>> =
-  IO.fx {
+fun <F> MonadError<F, Throwable>.getLineLengthAverage(path: FilePath): Kind<F, List<String>> =
```

Comment:
The code below supposedly reads from a file which should at least be `MonadDefer` I think. I don't think it matters in an example for `monadError` comprehensions but maybe an example without side-effects might be better.

Reply:
I've added async in the middle